libpod/container_log.go

```diff
@@ -72,5 +73,28 @@ func (c *Container) readFromLogFile(options *logs.LogOptions, logChannel chan *l
 		}
 		options.WaitGroup.Done()
 	}()
+	// Check if container is still running or paused
+	go func() {
```

Comment:
Will this also need to be added to WaitGroup?

Reply:
@QiWang19 When `t.Stop()` is called is should be unblocking the `t.Lines()` called in the other subroutine which would allow the other subroutine to end which calls `options.WaitGroup.Done()` before it returns.